docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,221 @@
+
+## Heroku
+
+### What is Heroku?
+Heroku is a cloud platform that allows developers to deploy their applications quick, easy, and free. Heroku is configured to handle a variety of programming languages and frameworks such as Java, Javascript, Python, Ruby, etc. For our case, Heroku is a great choice. Heroku works by having developers upload their project and Heroku runs our application for us and connects it to a URL waiting to respond to requests. In traditional cloud deployment platforms, developers have to create a virtual machine with a certain Operating System and hardware capabilities and then deploy their application within the application on the virtual machine and then connect it to a URL. A virtual machine can be envisioned as a computer that is separate from our physical computer. In this case, the virtual machine sits in some lab that the cloud provider your using owns, filled with hundreds of other VM's and you basically "rent" these machines. This is a complicated and long process. Thankfully, Heroku simplifies the process. All developers have to do is connect their code with Heroku and Heroku is in charge of doing the rest. 
+
+### Why do we use Heroku
+The main reason we're going to use Heroku is because it is easier to use than traditional cloud deployment platforms, as previously mentioned, and it allows us to deploy our application for free. When you deploy your application you need some computing resources. These computing resources cost money, therefore just to run a very simple server with minimum hardware capabilities can cost around 5-10 dollars a month. Heroku allows us to use these minimum computing resources for free. Of course if you want to upgrade your computing resources to use more RAM or CPU you can pay for that option.  
+
+### Install Heroku
+The way our application is structured, our frontend and backend are separated. They are not dependent on one another and are developed in different environments and servers. Therefore, we have to deploy our frontend and backend separately. Thankfully, Heroku provides great documentation as to how to deploy our React frontend application and our Java backend. However, before you deploy any application on Heroku we need to install Heroku on our machine. First, we need to create an account on Heroku. The link to sign up is https://signup.heroku.com/. Once you have signed up for a Heroku account, you have to install it on your machine. Because there are different installation methods depending on the machine that you are using, the download instructions for every operating system can be found here https://devcenter.heroku.com/articles/heroku-cli. What you are installing is Heroku's command line application which will allow you to open a terminal and run heroku commands on the terminal.
```

Comment:
```suggestion
The way our application is structured, our frontend and backend are separated. They are not dependent on one another and are developed in different environments and tech stacks. Therefore, we have to deploy our frontend and backend separately. Thankfully, Heroku provides great documentation as to how to deploy our React frontend application and our Java backend. However, before you deploy any application on Heroku, we'll need to install Heroku on our machine. First, we need to create an account on Heroku. The link to sign up is [signup.heroku.com](https://signup.heroku.com/). Once you have signed up for a Heroku account, you have to install some of its software on your machine. Because there are different installation methods depending on the machine that you are using, the download instructions for every operating system can be found [here](https://devcenter.heroku.com/articles/heroku-cli). You are installing Heroku's command line application, which will allow you to open a terminal and run Heroku commands on the terminal.
```